### PR TITLE
delayTS formatting, performance dashboard

### PR DESCRIPTION
**Pull request by TimCsaky**
# Description

- make delayTS in history table show in a human readable format
- add prod Performance Dashboard page
- update home page text

this is just making it look better for the sprint demo.
mail merge stuff is still in the works.

## Types of changes

New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/frontend/src/components/ches/HistoryTable.vue [hunk 1]
```diff
@@ -16,6 +16,14 @@
       :items="tableData"
       :search="search"
     >
+      <template #[`item.tag`]="{ item }">
+        <span>{{ !item.tag ? '-' : item.tag }}</span>
```
Comment (jujaga) [inline app/frontend/src/components/ches/HistoryTable.vue:20]: Normally if you don't need to negate a conditional, it's easier to read:
```js
<span>{{ item.tag ? item.tag : '-'  }}</span>
```

**Comment by bcgov-nr-csst** [comment]
PR Deployed at: https://chess-dev.apps.silver.devops.gov.bc.ca/pr-4

File: app/frontend/src/router/index.js [hunk 2]
```diff
@@ -50,6 +50,11 @@ export default function getRouter(basePath = '/') {
           requiresAuth: true
         }
       },
+      {
+        path: 'performance',
+        name: 'Performance',
+        component: () => import(/* webpackChunkName: "performance" */ '@/views/Performance.vue')
+      },
```
Comment (jujaga) [inline app/frontend/src/router/index.js:57]:
```js
        meta: {
          hasLogin: true
        }
```

**Review by jujaga — APPROVED** [review]

**Comment by bcgov-nr-csst** [comment]
PR Deployed at: https://chess-dev.apps.silver.devops.gov.bc.ca/pr-4

**Pull request merged**